### DISCUSSION TITLE
Can't get it to build

### DISCUSSION THREAD

**Discussion by nsheneman3** [SoCuul/SCInsta#59] [Support] [answered]
I believe I've done all of the prior steps, but when I get to the "Run Build Script" section, I'm lost.

I copy and paste the 
`$ chmod +x build.sh
$ ./build.sh <sideload/rootless/rootful>`

into terminal and it returns
`zsh: parse error near `\n'`

Anyone know what I could be doing wrong?

**Comment by SoCuul** [comment]
Did you copy the $ as well?
That's just meant as a placeholder for your prompt (what comes before the text you type in).

Try copying everything after the $

**Reply by nsheneman3** [reply]
Tried that. I was copying over both lines at once, but just tried individually and the first line didn't seem to do anything, and then the same _zsh: parse error near `\n'_ for the second line

**Reply by SoCuul** [reply]
The first line shouldn't do anything visually. That's how it's supposed to work.

**Reply by SoCuul** [reply, marked as answer]
Ah i think i see your problem. The 2nd command has the <> with multiple choices inside as placeholders.

You'll have to do `./build.sh sideload`

**Reply by nsheneman3** [reply]
That looks like it's working now, thank you!

**Reply by SoCuul** [reply]
Glad it's working now!
